src/com/puppetlabs/puppetdb/scf/migrate.clj

```diff
@@ -911,7 +1014,11 @@
    24 add-producer-timestamps
    25 structured-facts
    26 structured-facts-deferrable-constraints
-   27 switch-value-string-index-to-gin})
+   27 switch-value-string-index-to-gin
+   28 #(let [time-msg (with-out-str
+                        (time (apply lift-fact-paths-into-facts %&)))]
+         (log/info (str "lift: " time-msg)))
```

Comment:
@wyatt sure wrt "lift:", I was planning on removing it altogether in the final version, until @senior suggested we add it everwhere: https://tickets.puppetlabs.com/browse/PDB-1320